src/main/java/attendance/view/CustomStringBuilder.java

```diff
@@ -0,0 +1,13 @@
+package attendance.view;
+
+public class CustomStringBuilder {
+    private final StringBuilder stringBuilder = new StringBuilder();
+
+    public void appendLine(String content) {
+        stringBuilder.append(content).append('\n');
```

Comment:
```suggestion
        stringBuilder.append(content).append(System.lineSeparator());
```

`System.lineSeparator()` 를 사용했을 때 어떤 이점이 있을까요?

Reply:
시스템에 따라서 사용하는 개행문자가 다르기 때문에 사용하는군요!

- 문자열 사이에 함부로 escape 기호인 \n을 사용하는 것은 다른 종류의 OS에서 예상하지 못한 동작이 될 수 있기 때문입니다.
- 자바에서 대부분의 입출력API의 경우 \n을 자동으로 OS에 맞게 변환해주지만, Windows환경에서 일부 프로그램의 경우 줄바꿈이 적용이 되지 않을 수 있습니다. (예: 코드 에디터가 아닌 다른 프로그램 등)